src/crimson/os/seastore/transaction_manager.h

```diff
@@ -284,7 +284,7 @@ class TransactionManager : public ExtentCallbackInterface {
   /// Remove refcount for ref
   ref_ret dec_ref(
     Transaction &t,
-    LogicalCachedExtentRef &ref);
+    LogicalCachedExtentRef ref);
```

Comment:
The original `&` can save the effort to increment the reference counter, although the overhead is minor.